### DISCUSSION TITLE
Your multiplication button issue

### DISCUSSION THREAD

**Discussion by markagold1** [warmachine028/Better-Calculator#13] [Ideas]
Your multiplication button is broken because you have the wrong character string in your replaced list defined in function replace_().

The first item in the list should be chr(215), the code for the multiplication sign, but you have the lower case 'x' or chr(120).

Changing this item from chr(120) to chr(215) should fix your issue.

**Comment by warmachine028** [comment]
Thanks ;) . Would surely look into this
